src/status_im/multiaccounts/create/core.cljs

```diff
@@ -163,18 +163,22 @@
         name (gfycat/generate-gfy public-key)
         photo-path (identicon/identicon public-key)
         account-data {:name name :address address :photo-path photo-path}
-        new-multiaccount       {:address         address
-                                :name            name
-                                :public-key      public-key
-                                :photo-path      photo-path
-                                :latest-derived-path 0
-                                :accounts [wallet-account]
-                                :installation-id (random-guid-generator)
-                                :signing-phrase  signing-phrase
-                                :mnemonic        mnemonic
-                                :settings        constants/default-multiaccount-settings
-                                :network           constants/default-network
-                                :networks/networks constants/default-networks}
+        new-multiaccount {:address              address
+                          :name                 name
+                          :public-key           public-key
+                          :photo-path           photo-path
+                          :latest-derived-path  0
+                          :accounts             [wallet-account]
+                          :installation-id      (random-guid-generator)
+                          :signing-phrase       signing-phrase
+                          :mnemonic             mnemonic
+                          :settings             constants/default-multiaccount-settings
+                          :network              constants/default-network
+                          :networks/networks    constants/default-networks
+                          :keycard-instance-uid keycard-instance-uid
```

Comment:
could it be made conditional so that the keycard stuff is only assoced in db if there is a keycard

Reply:
done